api/graphql/reservation_units/reservation_unit_types.py

```diff
@@ -303,6 +303,7 @@ class ReservationUnitType(AuthNode, PrimaryKeyObjectType):
     buffer_time_before = Duration()
     buffer_time_after = Duration()
     metadata_set = graphene.Field(ReservationMetadataSetType)
+    state = graphene.String()
```

Comment:
This would be nice to get to the api as enum so that the values can be found in doc. But we don't have the "standardized" way of doing that yet so its fine as is.
It could be graphene.Enum type of field which *should* work with python enums (I have tested without luck :( ) Then again when have the field in the model as charfield and it has choices defined it will work _out-of-the-box_

Reply:
I'll merge this now and let's figure out a good way to handle those enums. I personally don't like to use "magic strings" so if there is a good way to use enums, I'd like to use that 😄